packages/loader/container-loader/src/containerContext.ts

```diff
@@ -170,6 +172,44 @@ export class ContainerContext implements IContainerContext {
 
 	private readonly _fluidModuleP: Promise<IFluidModuleWithDetails>;
 
+	/**
+	 * {@inheritDoc @fluidframework/container-definitions#IContainerContext.getEntryPoint}
+	 */
+	public getEntryPoint?(): Promise<FluidObject | undefined> {
+		if (this._disposed) {
+			throw new UsageError("The context is already disposed");
+		}
+		if (this._runtime !== undefined) {
+			// TODO: I don't get why this line complains about unsafe return, this._runtime.getEntryPoint has a return type
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+			return this._runtime?.getEntryPoint !== undefined
+				? this._runtime.getEntryPoint()
+				: Promise.resolve(undefined);
```

Comment:
there is no need to use Promise.resolve, just return undefined. the function is already async so will wrap return values in a promise. might fix your lint

Reply:
this can all be simplified to just return `this._runtime.getEntryPoint?.()`

Reply:
Oh... the simplification actually didn't work on its own because I hadn't declared the function `async`, it just returned a `Promise` but only `async` ones wrap automatically. Not sure why I didn't do that in the first place; simplified now :). Thanks!